src/utils/mod.rs

```diff
@@ -607,3 +607,65 @@ pub fn is_expn_of(cx: &LateContext, mut span: Span, name: &str) -> Option<Span>
         }
     }
 }
+
+// returns index after first CamelCase component of string
```

Comment:
Small pedantic nit: Should be a *rustdoc* `///` comment and the convention seems to be full sentences with a capital letter and dot.